crates/ruff_linter/src/rules/pyupgrade/rules/outdated_version_block.rs

```diff
@@ -256,11 +256,11 @@ fn compare_version(
             Ok(if or_equal {
                 // Ex) `sys.version_info <= 3.8`. If Python 3.8 is the minimum supported version,
                 // the condition won't always evaluate to `false`, so we want to return `false`.
-                if_minor < py_minor
+                if_minor <= py_minor
```

Comment:
I feel like we need different tie-breaking logic here depending on whether it's a great-than comparison or a less-than comparison, because this condition seems _correct_ (and the comment does too) for less-than.

Reply:
Yeah this fix is incorrect — this is really hurting my brain haha